src/ApiHandler.ts

```diff
@@ -139,6 +125,69 @@ export default class ApiHandler {
 			}
 		}
 
+		const byteFee = api.consts.transactionPayment.transactionByteFee;
+		const extrinsicBaseWeight = api.consts.system.extrinsicBaseWeight;
+		const coefficients = api.consts.transactionPayment.weightToFee[0];
+		const coeffFrac = coefficients.coeff_frac;
+		const coeffInt = coefficients.coeff_integer;
+		
+		// TODO: generalize for terms that may have more degrees & may be negative
+		const degree = coefficients.degree;
+		const negative = coefficients.negative;
+
+		for (let idx = 0; idx < block.extrinsics.length; ++idx) {
+			const extrinsic = block.extrinsics[idx];
+			if (!extrinsic.isSigned) {
+				continue;
+			}
+
+			try {
+				if (api.runtimeVersion.specName.toString() === 'kusama') {
+					extrinsics[idx].info = await api.rpc.payment.queryInfo(extrinsic.toHex(), parentHash);
```

Comment:
```suggestion
					extrinsic.info = await api.rpc.payment.queryInfo(extrinsic.toHex(), parentHash);
```

I just assume everything is a reference. Not a {java, type, ...}script expert. Maybe you can make use of an continue here to safe one level of indentation for the rest of the function.

Reply:
Great suggestion about using `continue` to save indentation :pray: